backend/production-optimiser-api/src/main/java/it/polimi/productionoptimiserapi/controllers/ResultController.java

```diff
@@ -0,0 +1,41 @@
+package it.polimi.productionoptimiserapi.controllers;
+
+import it.polimi.productionoptimiserapi.dto.OptimizationResultDto;
+import it.polimi.productionoptimiserapi.services.OptimizationResultService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
```

Comment:
Can add `@RequestMapping("/api/results")` here so every other mapping can be made as a subpath (e.g. `@GetMapping("/dummy")`

Reply:
Done